_data/sites.json

```diff
@@ -1121,8 +1143,8 @@
 			},
 			"instructions": {
 				"en": [
-					"Ensure the <a href=\"https://github.com/wet-boew/GCWeb/releases\">version 13.9.0 or more recent of GCWeb</a> is implemented.",
-					"Ensure the <a href=\"../page-details/page-details-docs-en.html\">version 3.0 or more recent of the page details</a> is implemented.",
+					"Ensure the <a href=\"https://github.com/wet-boew/GCWeb/releases\">version 13.9.0 or newer of GCWeb</a> is implemented.",
+					"Ensure the <a href=\"../page-details/page-details-docs-en.html\">version 3.0 or newer of the page details</a> is implemented.",
```

Comment:
@Garneauma Can you confirm what text should be used? Thanks

Reply:
@duboisp "newer".

Reply:
Confirmed, no change required.